src/Poseidon/Utils.hs

```diff
@@ -31,6 +31,7 @@ data PoseidonException =
     | PoseidonRemoteJSONParsingException String -- ^ An exception to indicate failed remote info JSON parsing
     | PoseidonGenericException String -- ^ A catch-all for any other type of exception
     | PoseidonEmptyOutPacNameException -- ^ An exception to throw if the output package lacks a name
+    | PoseidonUnequalBaseDirException FilePath FilePath FilePath -- ^ An exception to throw if genotype data files don't share a common base directory
```

Comment:
as discussed, don't think we need this.

Reply:
Let's discuss this above. I tried to replace it, but discovered a reason why to enforce this.